static/initial_informed_consent.html

```diff
@@ -1,182 +1,268 @@
+          map users back to their informed consent and identify faulty installation.
+          </p>
+          <p class="list-group-item-text">
+          Once an experiment has completed the resulting data will be sent, along with the experiment ID
+          and user ID to the data management system. Once the data management system confirms receipt
+          of the data it will be removed from the client software. Personally identifiable data that will be
+          collected and transmitted will be your IP address. Experiment results will be encrypted to ensure
+          they are not observable in transit and this personally identifiable data (as well as the unique user
+          ID) will only be accessible to researchers at the University of Toronto.
+          </p>
+          <p class="list-group-item-text">
+          Whenever the client communicates with the experiment controller or the data management
+          system the data will be encrypted and may also be sent to a proxy (e.g., in Amazon EC2 data
+          centers) before being forwarded on to the target system. The proxy will not store the data and
+          will not contain the key required to decrypt any data it forwards.
+          </p>
+        </div>
+
+        <div class="list-group-item">
+          <h4 class="list-group-item-heading termsHeading">RISKS</h4>
+
+          <p class="list-group-item-text">
+          The following risks may occur as a result of you being in this study:
+          </p>
+          <div class="list-group">
+            <div class="list-group-item">
+              <p>Your ISP or network administrator may be able to see the list of websites being accessed
+              by the measurement software if they are monitoring the network. There is a risk that
+              accessing such websites and running these tests violates network policies.</p>
+            </div>
+          </div>
+
+          <p class="list-group-item-text">
+          The potential risks posed by conducted these tests is highly specific to the context in which the
+          tests take place. These risks are highly dependent on the domestic political and legal context in
+          your country. While it is challenging to definitively identify what the risks of conducting these
+          tests may be in your country, we have compiled a number of metrics to help inform you of the
+          potential risks in your country.
+          </p>
+          <div class="list-group">
+            <div class="list-group-item">
+              <p>
+              Freedom House "Freedom on the Net score” <br/>
+              http://www.freedomhouse.org/report/freedom-net/freedom-net-2012
+              </p>
+            </div>
+            <div class="list-group-item">
+              <p>
+              Economist "Democracy Index score"<br/>
+              http://www.eiu.com/public/thankyou_download.aspx?activity=download&campaignid=DemocracyIndex12
```

Comment:
What is the purpose of these links and can we remove them to ensure better privacy? The reason that we download all of the content to our server, then display what we downloaded is that we want to make sure that we aren't sending our users to the original source. If the user goes to the original source, then Freedom House or anyone else can have a list of our clients and we don't want that. Additionally, even if these are left in just for reference, there is still the potential that these links could identify the user due to browser prefetching (the browsing looking up the domains even though they are never visited)